web-gateway/conf/application.conf

```diff
@@ -2,5 +2,9 @@
 
 lagom.circuit-breaker.default.call-timeout = 5s
 
+online-auction.instruction.show = false
```

Comment:
I believe the java app defaults this to `true`, however, I've seen no traces of instructions in the scala implementation. Do we want to show the instructions?